benchmark-android/build.gradle

```diff
@@ -0,0 +1,61 @@
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath 'com.android.tools.build:gradle:2.3.3' // jcenter has the latest
```

Comment:
Should we define a version variable for this at the top-level? There are now 3 modules that use it.